--- a/drivers/net/wireless/bcm4336/dhd_config.c	2015-08-27 22:57:27.000000000 +0200
+++ b/drivers/net/wireless/bcm4336/dhd_config.c	2015-08-27 23:06:13.997638362 +0200
@@ -1059,6 +1059,7 @@
 	}
 #endif
 
+#ifdef DHD_DEBUG
 	/* Process dhd_console_ms */
 	memset(pick, 0, MAXSZ_BUF);
 	len_val = process_config_vars(bufp, len, pick, "dhd_console_ms=");
@@ -1066,6 +1067,7 @@
 		dhd_console_ms = (int)simple_strtol(pick, NULL, 0);
 		printf("%s: dhd_console_ms = 0x%X\n", __FUNCTION__, dhd_console_ms);
 	}
+#endif
 }
 
 void
